week1/gallery.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>숭이의 동물원</title>
+  </head>
+  <body>
+    <header id="title">🤍숭이의 동물원🤍</header>
+    <section id="preview">
+      <img src="./src/강아지1.jpeg" alt="강아지사진1" />
+      <img src="./src/고양이4.png" alt="고양이사진4" />
+      <img src="./src/숭이1.jpeg" alt="원숭이사진1" />
+      <img src="./src/강아지2.jpg" alt="강아지사진2" />
+      <img src="./src/고양이2.jpg" alt="고양이사진2" />
+      <img src="./src/숭이2.jpg" alt="원숭이사진2" />
+      <img src="./src/강아지3.jpg" alt="강아지사진3" />
+      <img src="./src/고양이3.jpeg" alt="고양이사진3" />
+      <img src="./src/숭이3.jpg" alt="원숭이사진3" />
+    </section>
+    <nav>
+      <p>클릭하면 이동해요</p>
+      <div>
+        <a href="#monkey">🐵 숭 🐵</a>
+        <a href="#dog">🐶 멍 🐶</a>
+        <a href="#cat">🐱 냥 🐱</a>
+      </div>
+    </nav>
```

Comment:
```suggestion
    <nav>
      <p>클릭하면 이동해요</p>
      <ul>
        <li><a href="#monkey">🐵 숭 🐵</a></li>
        <li><a href="#dog">🐶 멍 🐶</a></li>
        <li><a href="#cat">🐱 냥 🐱</a></li>
      </ul>
    </nav>
```
이렇게 하면 더 시맨틱해질 수 있을 것 같아용 ~! ㅎㅎ

Reply:
헉 이부분을 고려했었는데 시맨틱을 위해 기능적인 목적은 없는 중첩 태그를 사용하는게 맞나???라고 생각해서 우선순위인 a태그를 선택했었어요!! 근데 시맨틱을 위해서면 태그를 중첩해서 사용하는게 좋을까요?!